recipes/btllib/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
 
 build:
   skip: True  # [py < 38 or win]
```

Comment:
```suggestion
```
This could be removed since we don't build anything less than 3.8 any more and don't build Windows.